producers/mysql-producer/src/main/java/com/flipkart/aesop/runtime/producer/avro/MysqlAvroEventManager.java

```diff
@@ -75,11 +77,15 @@
 	/** physical source id */
 	protected final int pSourceId;
 
+	/** flag for getting old values */
+	private final boolean isOldValueRequired;
```

Comment:
No need of prefixing is. For boolean variables, just keep the part other than is. is will typically be used as part of the getter. So, just boolean oldValueRequired is fine.